src/bussinessLogic/ManageArray.java

```diff
@@ -37,11 +35,10 @@
      *
      * @param manageMap
      */
-    public ManageArray(MapManager manageMap, ConstantsInterface constant) {
+    public ManageArray(MapManager manageMap) {
         this.manageMap = manageMap;
-        this.constant = constant;
-        this. numberOfLetters = constant.getNumberOfLetters();
-        this.alphabetMultiplier = constant.getAlphabetMultiplier();
+        this. numberOfLetters = Constants.NUMBER_OF_LETTERS;
```

Comment:
`Constants.NUMBER_OF_LETTERS` can be used everywhere where there is `this.numberOfLetters`.

This is valid for `this.alphabetMultiplier` as well.

Reply:
👍